src/org/zaproxy/zap/extension/quickstart/resources/help/contents/quickstart.html

```diff
@@ -21,11 +21,30 @@
 	For a more in depth test you should explore your application using your browser or 
 	automated regression tests while proxying through ZAP.
 </p>
+<H2>Launch Browser</H2>
+As long as you have the Selenium add-on installed then you will see an option to launch browsers.<br>
+Select the browser you want from the pull-down and press the 'Launch Browser' button.<br>
+Browsers launched this way will be configured to proxy via ZAP and will ignore certificate errors,
+so you will not need to import the ZAP Root CA Certificate.<br>
```

Comment:
Are you getting cert errors?
We're using Selenium to control the browsers, and use this code: https://github.com/zaproxy/zap-extensions/blob/master/src/org/zaproxy/zap/extension/selenium/ExtensionSelenium.java#L424-L519 which instructs the various drivers to ignore cert errors.
I've just tried again with FX and Chrome in a new dir, completely new cert. Both show warnings in the tool bar but still work fine.
Try it and see :)